airflow/gcp/example_dags/example_bigquery.py

```diff
@@ -201,10 +201,10 @@
 
     create_view = BigQueryCreateEmptyTableOperator(
         task_id="create_view",
-        dataset_id=LOCATION_DATASET_NAME,
+        dataset_id=DATASET_NAME,
         table_id="test_view",
         view={
-            "query": "SELECT * FROM `{}.test_table`".format(DATASET_NAME),
+            "query": f"SELECT * FROM `{PROJECT_ID}.{DATASET_NAME}.test_table`",
```

Comment:
I checked it. We should use legacy SQL by default.  Unfortunately, this has been changed for unknown reasons. This change is not backward compatible and we should restore the original behavior. If we want this behavior, then we should introduce a transition period, which introduced warnings, because this is a very important service.

Reply:
So, someone changed BQ operators and broken compatibility?

Reply:
Yes. Someone did. We must restore the previous behavior.